src/libraries/System.Data.Common/src/System/Xml/DataDocumentXPathNavigator.cs

```diff
@@ -50,7 +47,7 @@ public override string Value
             get
             {
                 XPathNodeType xnt = _curNode.NodeType;
-                return xnt == XPathNodeType.Element || xnt == XPathNodeType.Root ? _curNode.InnerText : _curNode.Value;
+                return xnt == XPathNodeType.Element || xnt == XPathNodeType.Root ? _curNode.InnerText : _curNode.Value!;
```

Comment:
What's protecting _curNode.Value from being null? (here and below)

Reply:
Quite frankly I don't know. But I can't make the property nullable because the base definition isn't nullable. Changing the code return an empty string when `_curNode.Value` seems worse.

Reply:
> But I can't make the property nullable because the base definition isn't nullable

Does that mean the base is defined incorrectly?

Reply:
Looking through the code it seems like other overrides return an empty string instead, so it seems the intent was non-nullable.